README.md

```diff
@@ -78,6 +79,7 @@ The following volumes are available for mounting:
 | `/mnt/borg-repository` | Mount your borg backup repository here. |
 | `/etc/borgmatic.d` | Where you need to create crontab.txt and your borgmatic config.yml |
 | `/root/.borgmatic` | **Note** this is now redundant and has been deprecated, please remove this from your configs |
+| `/root/.local/state/borgmatic` | Here are the state files for periodic checks, prunes, and other things. |
```

Comment:
Note that the state files are not (currently) used for pruning at all.

Reply:
You're right. That is a mistake by me. Please forget/remove the pruning part/word.

Thanks

Reply:
Just made the change, but don't know if it's altered correctly in the PR...

Reply:
Yup, it looks like it's there!